src/views/lists_home.js

```diff
@@ -151,32 +160,68 @@ class Icons extends Component {
           {list2.map((l, i) => (
             <ListItem
               roundAvatar
-              avatar={{ uri: l.avatar_url }}
+              leftAvatar={{ source: { uri: l.avatar_url } }}
               key={i}
               onPress={log}
               title={l.name}
               subtitle={l.subtitle}
+              disclosure
+              bottomDivider
             />
           ))}
         </List>
         <List>
-          {
-            list2.map((l, i) => (
-              <ListItem
-                key={i}
-                leftIcon={{ name: 'user-circle-o', type: 'font-awesome', style: {color: 'blue'} }}
-                title={l.name}
-                titleStyle={{color: 'red'}}
-                subtitle={l.subtitle}
-                rightTitle='11:00am'
-                rightTitleStyle={{color: 'green'}}
-              />
-            ))
-          }
+          {list2.map((l, i) => (
+            <ListItem
+              key={i}
+              leftIcon={{
+                size: 40,
+                name: 'user-circle-o',
+                type: 'font-awesome',
+                color: 'blue',
+              }}
+              title={l.name}
+              titleProps={{ children: l.name, style: { color: 'red' } }}
+              subtitle={l.subtitle}
+              rightTitle="11:00am"
+              rightTitleProps={{ style: { color: 'green' } }}
+              bottomDivider
+            />
+          ))}
+        </List>
+        <List>
+          <ListItem
+            title="Name"
+            inputProps={{ placeholder: 'Type your name' }}
```

Comment:
should there be a prop to add `input`? or if you pass inputProps it adds the input on the ListItem component?

maybe use can add a `input={true}` prop along with `inputProps`

Reply:
I was thinking of that.

inputProps is to be more consistent with `titleProps` or `subtitleProps`. But TextInput doesn't take any values like title or subtitle (`title="Hello"`).

Specified `input` and `inputProps` is repetitive, so maybe just specified `input` that takes all the props like this: 
```js
input={{
  containerStyle={styles.container}
  onChange={() =>{}}
}}

Reply:
Yeah, I actually like that. With just 1 `input` prop.. you can all all the props within it. Looks organized to me and makes sense. Let's do that.